nucleus/deployment/common/src/main/java/fish/payara/deployment/util/GAVConvertor.java

```diff
@@ -171,18 +173,27 @@ private URL findArtefactURL(List<URL> repositoryURLs, String relativeURLString)
                 try {
                     artefactURL = new URL(repositoryURL, relativeURLString + archiveType);
 
-                    HttpURLConnection httpConnection = (HttpURLConnection) artefactURL.openConnection();
-                    
-                    String auth = artefactURL.getUserInfo();
-                    if (auth != null) {
-                        String encodedAuth = Base64.getEncoder().encodeToString(auth.getBytes());
-                        httpConnection.setRequestProperty("Authorization", "Basic " + encodedAuth);
+                    if ("file".equalsIgnoreCase(artefactURL.getProtocol())) {
+                        if (Files.exists(Paths.get(artefactURL.toURI()))) {
+                            validURLFound = true;
+                        }
+                    } else {
+                        HttpURLConnection httpConnection = (HttpURLConnection) artefactURL.openConnection();
```

Comment:
This makes some assumption about the type of connection but it might be a JarURLConnection. Like this one: jar:http://www.foo.com/bar/baz.jar

Reply:
I only needed a local repository but jar scheme support also possible

Reply:
This needs to be decided by someone from Payara.

Reply:
I'd say that's outside the scope of this PR. 
I'm not immediately sure of the need to handle them either tbh - I think that functionality is covered by the standard --deploy option unless you're talking about using a Jar Url as a "maven" repository in its own right, which sounds mega-niche!